libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -547,6 +566,25 @@ static IotMqttError_t _deserializeIncomingPacket( _mqttConnection_t * pMqttConne
 
     if( status != IOT_MQTT_SUCCESS )
     {
+        /* Coverity finds a USE_AFTER_FREE error at this line. This is a false positive.
+         *
+         * This error is triggered by passing a freed argument 'pMqttConnection'
+         * to 'IotLogError'. Coverity assumes that 'pMqttConnection' was freed in
+         * '_IotMqtt_CreateOperation', which was invoked from '_deserializePublish'
+         * invoked above, where cleanup code will free 'pNewMqttConnection' upon
```

Comment:
remove `invoked` at beginning of line

pNewMqttConnection is wrong variable name